### DISCUSSION TITLE
adding Button update 1.0.5 issue

### DISCUSSION THREAD

**Discussion by Mark-Longley** [zachowj/hass-node-red#105] [Q&A] [answered]
Hi, should this update allow us to create a new entity with Type:Button? 
If i've read it right and not misunderstood?
I'm trying to add entity node and select type 'Button' but I only have the Binary_Sensor, Sensor and Switch options available.

Many Thanks
Mark

**Comment by zachowj** [comment, marked as answer]
It's a new node called button under a new category called home assistant entities

**Reply by Mark-Longley** [reply]
I needed to update the pallet to see it. Works a treat so far, great work, thanks for the support !
